WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -226,9 +228,15 @@ public boolean onOptionsItemSelected(final MenuItem item) {
 
             if (peopleInviteFragment == null) {
                 Blog blog = WordPress.getCurrentBlog();
-                peopleInviteFragment = PeopleInviteFragment.newInstance(blog.getDotComBlogId());
+                String dotComBlogId = blog.getDotComBlogId();
+                if (dotComBlogId != null && !dotComBlogId.isEmpty()) {
+                    peopleInviteFragment = PeopleInviteFragment.newInstance(dotComBlogId);
+                } else {
+                    AppLog.e(T.PEOPLE, "getDotComBlogId() returned null or empty string!");
```

Comment:
Assuming that blog is not null here, could we please log some info about the current blog? Like the URL of it, or any other info you think it's relevant to debug this further.